### ISSUE TITLE
Results & ckpts of different sized Latte on UCF-101

### ISSUE THREAD

**Issue by yizenghan** [Vchitect/Latte#101]
Hi there, could you report the FVD/IS results of different-sized Latte models on UCF-101? 

If possible, the pre-trained checkpoints would be useful. Thanks!

**Comment by maxin-cn** [comment]
We did not train models of different sizes on UCF101. We have only trained models of different sizes on FFS. You can find pre-trained checkpoints at [here](https://huggingface.co/maxin-cn/Latte/tree/main).

**Comment by yizenghan** [comment]
Hi, I'm here again. A small question: did you notice the difference of the loss value between training by videos only and training by videos together with images?

**Comment by yizenghan** [comment]
Hi, may I ask about the number of training iters of the released ckpts? In the paper I found the plots illustrate training 150k iters. In the code I find the max training steps are 1e6.

**Comment by maxin-cn** [comment]
> Hi, may I ask about the number of training iters of the released ckpts? In the paper I found the plots illustrate training 150k iters. In the code I find the max training steps are 1e6.

I kind of forget exactly how many iterations. But according to the results of other people's replications, it's about 8
A100 (80G) GPU training for one week can achieve the value reported in the paper.
